README.md

```diff
@@ -28,7 +28,13 @@ After integrating dcurl into IRI, performance of [attachToTangle](https://iota.r
 
 ## IRI Adaptation
 [Modified IRI accepting external PoW Library](https://github.com/DLTcollab/iri)
-Supported IRI version: 1.6.0
+Supported IRI version: 1.7.0
```

Comment:
Can we provide backward JNI support?

Reply:
Like support the old way to copy the `libdcurl.so` file and assign the correct path?
If so, it is still supported in IRI.
I just mark it as deprecated in the `README.md` since I guess it might be replaced by the new loading method.